rindexer_rust_playground/src/rindexer_lib/typings/rindexer_playground/mod.rs

```diff
@@ -1,7 +1,2 @@
 #![allow(dead_code, unused)]
-
-/// THIS IS A GENERATED FILE. DO NOT MODIFY MANUALLY.
```

Comment:
how come we lose this being wrote to the files?

Reply:
No idea, just regenerated and nothing else was changed so maybe a change many versions ago?